src/pages/rooms/room-creation/room-creation.page.jsx

```diff
@@ -1,9 +1,180 @@
+    setCategory(e.target.value);
+    console.log(e);
+  };
+
+  const getAllCategories = () => {
+    authenticationService
+      .get(Endpoints.Category)
+      .then((result) => {
+        setCategories(result.data);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  };
+
+  useEffect(() => {
+    hubConnection.start().then(() => {
+      console.log("connected");
+      getAllCategories();
+
+      hubConnection.on("All categories uploaded", () => {
+        getAllCategories();
+      });
+    });
+  }, []);
+
+  return (
+    <div className="qcg-room-creation-page qcg-flex qcg-flex-center">
+      <div className="wrapper">
+        <div className="creation-wrapper qcg-flex qcg-flex-column qcg-flex-align-center">
+          <h1 className="qcg-flex qcg-flex-center">Create Your Room</h1>
+          <div className="create-button qcg-flex">
+            <Button
+              variant="contained"
+              endIcon={<ArrowBackIcon />}
+              onClick={() => navigationService(States.RoomsList)}
+            >
+              back to rooms list
+            </Button>
+          </div>
+          <Box
+            component="form"
+            sx={{
+              "& > :not(style)": { m: 1, width: "25ch" },
+            }}
+            noValidate
+            autoComplete="off"
+          >
+            <TextField
+              id="standard-basic"
+              label="Room Name"
+              variant="standard"
+              onChange={(e) => {
+                setRoomName(e.target.value);
+              }}
+            />
+            <br />
+            <FormControl variant="standard" sx={{ m: 1, minWidth: 120 }}>
+              <InputLabel id="demo-simple-select-standard-label">
+                Category
+              </InputLabel>
+              <Select
+                labelId="demo-simple-select-standard-label"
+                id="demo-simple-select-standard"
+                value={category}
+                onChange={handleChange}
+                label="Category"
+              >
+                {categories.length !== 0 ? (
+                  categories.map((e) => {
+                    return <MenuItem value={e._id}>{e.categoryName}</MenuItem>;
+                  })
+                ) : (
+                  <></>
+                )}
+              </Select>
+            </FormControl>
+
+            <div className="qcg-flex qcg-flex-center">
+              <FormControl>
+                <FormLabel id="creation-page-radio-buttons-group-label">
+                  Status
+                </FormLabel>
+                <RadioGroup
+                  row
+                  aria-labelledby="creation-page-radio-buttons-group-label"
+                  name="row-radio-buttons-group"
+                >
+                  <div className="qcg-flex qcg-flex-center">
+                    <FormControlLabel
+                      value="public"
+                      control={<Radio />}
```

Comment:
change from <Radio /> to <Radio checked={isPublic} /> so it always starts as checked for the public status.